README.md

```diff
@@ -231,6 +232,16 @@ most recent successful snapshot:
 Note: the `GHE_DATA_DIR` variable set in `backup.config` can be used to change
 the disk location where snapshots are written.
 
+### How do backup snapshots differ from the HA (High Availability) replica?
+The backup utilities and [HA replica](https://help.github.com/enterprise/admin/guides/installation/high-availability-cluster-configuration/) serve different yet complementary purposes as part of a recommended GitHub Enterprise deployment.
```

Comment:
`It is recommended that both backup utilities and a HA replica are used as part of a GitHub Enterprise deployment but they serve different roles.` :question: